### PR TITLE
Allow pod installation with rubygems

### PR DESCRIPTION
**Pull request by mgrant34**
The latest pods release is 1.5.3. This project is on 1.4.0.

Rather than all users who fork the repo having to revert their system pods installation to 1.4.0, this PR will allow users to install pods 1.4.0 only in the nano-wallet directory by using `bundle exec pod install`

### CODE REVIEW

File: Pods/Pods.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		033222A97BF88FE2C6045E2AFED3E6C1 /* EFQRCode.swift in Sources */ = {isa = PBXBuildFile; fileRef = 17CE706E994A283CFFD051E0393D038A /* EFQRCode.swift */; settings = {COMPILER_FLAGS = "-w -Xanalyzer -analyzer-disable-all-checks"; }; };
 		035AFF4345B3E0909DE020BEE97F80EB /* UIScrollView.swift in Sources */ = {isa = PBXBuildFile; fileRef = A2832BEF6C1887F6F2B4445D36E7034B /* UIScrollView.swift */; settings = {COMPILER_FLAGS = "-w -Xanalyzer -analyzer-disable-all-checks"; }; };
 		0444ADBF79FB2F48EA3F5A8879D22804 /* View.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8A303CEC0832E2939F93AF0382B9DD48 /* View.swift */; settings = {COMPILER_FLAGS = "-w -Xanalyzer -analyzer-disable-all-checks"; }; };
-		052A2A8B80AF8F76C2BFD58614DE9856 /* EFQRCode-dummy.m in Sources */ = {isa = PBXBuildFile; fileRef = 4AD5A816197B84E2BDF02830102CA534 /* EFQRCode-dummy.m */; };
+		052A2A8B80AF8F76C2BFD58614DE9856 /* EFQRCode-dummy.m in Sources */ = {isa = PBXBuildFile; fileRef = 1C118E8E07C44E3A5C4182A4F102BAA0 /* EFQRCode-dummy.m */; };
```
Comment (mgrant34) [inline Pods/Pods.xcodeproj/project.pbxproj]: this entire `Pods/` directory prob shouldn't be in history

**Comment by zackshapiro** [comment]
There's nothing sacred about 1.4.0 fwiw. I like to commit the pods file so someone can pull it and go right away without an internet connection although it's not that important

**Comment by mgrant34** [comment]
I think its a good practice to use a gem file, given each pods update should undergo at least a quick review/qa so it helps avoid a pods update causing a delay in development/releases. Plus in the team environment there is no forcing everyone to change their system pods version. Everyone just runs the same command to install pods. 

And totally re: internet connection, although it took a few minutes to clone the repo. I'd assume that was because of the pods project. Wouldnt it take just as long to re-run pod install?

**Comment by bibobode** [comment]
Thank you so much for your contribution! Your changes have been incorporated into https://github.com/nano-wallet-company/nano-wallet-ios/pull/20

**Pull request closed without merging**